docs/source/getting_started/platform.rst

```diff
@@ -302,3 +302,150 @@ account (top right)-> preferences on the
 .. warning:: Keep the token for yourself and don't share it. Anyone with the
           token could access your datasets!
 
+
+How to use S3 with lightly
+------------------------------
+
+
+Lightly allows you to configure a remote datasource like Amazon S3 (Amazon Simple Storage Service) so that you don't need to upload your data to Lightly and can preserve its privacy.
+
+
+**What you will learn**
+
+
+In this guide we will show you how to setup your S3 bucket, configure your dataset to use said bucket and to only upload metadata to lightly while preserving the privacy of your data
+
+
+Setting up Amazon S3
+^^^^^^^^^^^^^^^^^^^^^^
+For Lightly to be able to create so-called `presigned URLs/read URLs <https://docs.aws.amazon.com/AmazonS3/latest/userguide/ShareObjectPreSignedURL.html>`_ to be used for displaying your data in your browser, Lightly needs at minimum to be able to read and list permissions on your bucket. If you want Lightly to create optimal thumbnails for you while uploading the metadata of your images, write permissions are also needed.
```

Comment:
optimal thumbnails? what do you mean?

Reply:
optional?

Reply:
optimal in a sense that we know best how to make good thumbnails (size, compression, etc)

Reply:
I still don't think optimal makes any sense here 😄 but that's a minor thingy so you decide...